src/components/popover/__stories__/index.story.tsx

```diff
@@ -0,0 +1,183 @@
+import * as React from 'react';
+import { Text, Button, Popover, Avatar } from '@/index';
+import { PositionType } from '../Popover';
+
```

Comment:
Create one story each on how to use `updateOptions ` & `dismissOptions` prop

Reply:
updateOptions story testing